This patch adds the --disable-doc parameter to configure to allow to not
build the API documentation if not requested, not only if doxygen is not
found (as the API documentation takes a bit of time).

Index: flac-1.1.2/configure.in
===================================================================
--- flac-1.1.2.orig/configure.in
+++ flac-1.1.2/configure.in
@@ -181,7 +181,13 @@ dnl check for i18n(internationalization)
 AM_ICONV
 AM_LANGINFO_CODESET
 
-AC_CHECK_PROGS(DOXYGEN, doxygen)
+AC_ARG_ENABLE([doc],
+	AC_HELP_STRING([--disable-doc], [Disable API documentation building via Doxygen]))
+
+if "x$enable_doc" != "xno"; then
+  AC_CHECK_PROGS(DOXYGEN, doxygen)
+fi
+
 AM_CONDITIONAL(FLaC__HAS_DOXYGEN, test -n "$DOXYGEN")
 if test -n "$DOXYGEN" ; then
 AC_DEFINE(FLAC__HAS_DOXYGEN)
